src/cli/build.js

```diff
@@ -0,0 +1,20 @@
+const fs = require('fs');
+const path = require('path');
+const { execSync } = require('child_process');
+
+module.exports = (argv) => {
+  const dockerfile = path.resolve(__dirname, '../../Dockerfile');
+  const [head, tail = 'develop'] = argv.tag.split(':');
+
+  if (!fs.existsSync('./build')) fs.mkdirSync('./build');
```

Comment:
Not really a concern right now, but with explicit paths here this will not work on Windows, right?

Reply:
why wouldn't it? wouldn't the node runtime normalize the path?

Reply:
Right, I meant to comment on the `docker save -o build/...` below, then thought about this and somehow left the comment here... And I think there were other `exec` calls too that included paths.

Reply:
right.. those could potentially fails.. not sure.. let's worry about them later when needed.. :see_no_evil: